packages/shorebird_cli/lib/src/commands/patch/patch_ios_framework_command.dart

```diff
@@ -194,18 +211,73 @@ Please re-run the release command for this version or create a new release.''');
       return ExitCode.software.code;
     }
 
+    // TODO use linker if available
```

Comment:
Does this break linking?

Reply:
Resolved offline, this just means this command hasn't implemented support for linking with aot-tools yet. Filed https://github.com/shorebirdtech/shorebird/issues/1640 to track